lib/common/output.c

```diff
@@ -141,7 +141,7 @@ pcmk__register_messages(pcmk__output_t *out, pcmk__message_entry_t *table) {
     pcmk__message_entry_t *entry;
 
     for (entry = table; entry->message_id != NULL; entry++) {
-        if (safe_str_eq(out->fmt_name, entry->fmt_name)) {
+        if (pcmk__str_any_of(entry->fmt_name, "default", out->fmt_name, NULL)) {
```

Comment:
Is "default" guaranteed to be processed first somehow, so overrides work as expected?

Reply:
Not at the moment, no.  It currently depends on the order things are listed in a pcmk__message_entry_t table.  An easy way to guarantee this would be to process the table in two steps - first, iterate looking for defaults.  Second, iterate adding everything else.  There's probably no performance worries there because I can't see these tables ever growing so large that it becomes a problem.

Reply:
Yep I just realized that looking at it some more. I think it's fine as is, but maybe make that more explicit in the comment.